src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length >= 2) {
+      stylesObject[part[0].trim()] = part[1].trim();
```

Comment:
The code does not handle the case where a CSS property or value contains leading or trailing whitespace. It's important to trim both the property name and the value to ensure the object is created correctly.